src/poliastro/patched_conics.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+"""Patched Conics Computations
+
+Contains methods to compute interplanetary trajectories approximating the three
+body problem with Patched Conics.
+"""
+
+from astropy import units as u
+from poliastro.twobody import Orbit
+from poliastro.constants import J2000
+
+
+@u.quantity_input(a=u.m)
+def compute_soi(body, a=None):
+    """Computes the approximated radius of the Laplace Sphere of Influence (SOI)
```

Comment:
I would remove the class from here, and add the `Parameters` section.

Reply:
I am not sure but, do you mean to pass to the function the different parameters instead of the whole object?

Reply:
I mean, having a shorter summary in the first line of the docstring and adding the "parameters" section as explained here:

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections

Reply:
It should be done now 👍